templates/advanced/api/services/Ember.js

```diff
@@ -66,13 +67,14 @@ var Ember = {
 			var links = {};
 
 			_.each( associations, function ( assoc ) {
+				var assocModelIdentifier = pluralize( _.kebabCase( sails.models[assoc.collection || assoc.model].globalId ) );
 				var assocName = assoc.type === "collection" ? pluralize( assoc.collection ) : pluralize( assoc.model );
```

Comment:
No Problem, I'm just adding my notes while I'm looking through this and testing.